<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>533</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="LabelNumDept">
        <property name="text">
         <string>Donnez un numéro de département</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditNumDept"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonNomDept">
        <property name="text">
         <string>Avoir le nom du département</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="labelNomDept">
      <property name="text">
       <string>Nom du département</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>498</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QComboBox" name="comboBoxRegions">
      <item>
       <property name="text">
        <string>Choisissez une région</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBoxDepartements">
      <item>
       <property name="text">
        <string>Choisissez un département</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBoxVilles">
      <item>
       <property name="text">
        <string>Choisissez une ville</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
